packages/app/src/components/catalog/component.tsx

```diff
@@ -191,6 +192,9 @@ const component = (
         </Grid>
       </Grid>
     </EntityLayout.Route>
+    <EntityLayout.Route path="/Alerts" title="Alerts">
```

Comment:
Path should be lowercase, and I'd prefer to have a generic `Monitoring` tab, instead of `Alerts`.

Reply:
Fixed in the latest commit